multiapps-mta/src/main/java/org/cloudfoundry/multiapps/mta/util/InflatorUtil.java

```diff
@@ -0,0 +1,52 @@
+package org.cloudfoundry.multiapps.mta.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.function.ObjIntConsumer;
+import java.util.zip.DataFormatException;
+import java.util.zip.Inflater;
+
+import org.cloudfoundry.multiapps.common.ContentException;
+import org.cloudfoundry.multiapps.common.SLException;
+import org.cloudfoundry.multiapps.mta.Messages;
+
+public class InflatorUtil {
+
+    private static final int BUFFER_SIZE = 4 * 1024; // 4KB
+
+    private InflatorUtil() {
+
+    }
+
+    public static void inflate(EntryToInflate entryToInflate, ObjIntConsumer<byte[]> decompressedBytesConsumer) {
```

Comment:
can you separate this method in smaller ones

Reply:
+1

Reply:
done